utils/manifest-utils/src/keystore/mod.rs

```diff
@@ -0,0 +1,301 @@
+pub mod golem_keystore;
```

Comment:
`keystore.rs` with `keystore` folder will be more readable

Reply:
Moved it to `keystore.rs`, but initially place it on purpose in `mod.rs`. IDEs tree view do not display `keystore` dir next to `keystore.rs`, which for me is less readable.